source/Ship.cpp

```diff
@@ -482,6 +482,27 @@ void Ship::Load(const DataNode &node)
 				leak.closePeriod = child.Value(3);
 			leaks.push_back(leak);
 		}
+		else if(key == "auto explosion")
+		{
+			if(child.Size() >= 2)
+				autoExplosion.baseMult = child.Value(1);
+			else
+				autoExplosion.baseMult = 1.;
```

Comment:
```suggestion
			autoExplosion.baseMult = child.Size() >= 2 ? child.Value(1) : 1.;
```
just my personal preference

Reply:
Mine too, no idea why I didn't